### ISSUE TITLE
js error

### ISSUE THREAD

**Issue by Starkguava** [yining1023/pix2pix_tensorflowjs#6]
tfjs:formatted:7997 Uncaught (in promise) Error: Batch normalization gradient requires mean and offset to have equal ranks.
    at lv (tfjs:formatted:7997)
    at batchNorm_ (tfjs:formatted:13935)
    at Object.batchNorm__op [as batchNorm] (tfjs:formatted:10410)
    at batchnorm (pix2pix.js:34)
    at pix2pix.js:64
    at tfjs:formatted:9567
    at e.t.scopedRun (tfjs:formatted:9576)
    at e.t.tidy (tfjs:formatted:9560)
    at Object.dx [as tidy] (tfjs:formatted:13197)
    at Pix2pix.transfer (pix2pix.js:47)

i use my model which is 54.4MB. how can i run it in server?

**Comment by Heath123** [comment]
![image](https://user-images.githubusercontent.com/13787163/104196032-3f6eac80-541b-11eb-8779-0ca620e8dc40.png)

**Comment by Heath123** [comment]
I get this too on the demo @yining1023

**Comment by yining1023** [comment]
@Starkguava @Heath123 thank you so much for reporting this issue!

The issue should be fixed now, you can check out the demo here: https://yining1023.github.io/pix2pix_tensorflowjs/
